### DISCUSSION TITLE
jcvi.graphics.karyotype file output name

### DISCUSSION THREAD

**Discussion by aersoares81** [tanghaibao/jcvi#499] [Ideas]
Suggestion: Let the user choose `python -m jcvi.graphics.karyotype seqids layout` output file name instead of defaulting to karyotype.pdf.

**Comment by tanghaibao** [comment]
@aersoares81 

Fixed in https://github.com/tanghaibao/jcvi/pull/502

Use `-o output.pdf` to specify an output file.
